apps/web/graphql/client.ts

```diff
@@ -43,6 +47,14 @@ function create(initialState?: any) {
     ssrMode: !isBrowser, // Disables forceFetch on the server (so queries are only run once)
     link: httpLink,
     cache: new InMemoryCache({
+      dataIdFromObject(obj: IdGetterObj) {
+        switch (obj.__typename) {
+          case 'ArticleCategory':
```

Comment:
It should be safe to make this more generic as in always return the id with locale appended.
Maybe remove the switch and return defaultDataIfFromObject with locale appended to it?